SMDS_3/doc/SMDS_3/Concepts/MeshComplex_3InTriangulation_3.h

```diff
@@ -46,7 +46,7 @@ The data structure encodes the final mesh at the end of the meshing process.
 
 \sa `MeshDomain_3`
 \sa `MeshComplexWithFeatures_3InTriangulation_3`
-\sa `CGAL::make_mesh_3()`
+\sa \link make_mesh_3() `CGAL::make_mesh_3()`\endlink
 
```

Comment:
Most likely this is the place where the overnight build warning:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-3/doc/SMDS_3/Concepts/MeshComplex_3InTriangulation_3.h:40: warning: unable to resolve link to 'make_mesh_3()' for \link command
```
refers to , code should most likely be:
```
\sa \link CGAL::make_mesh_3() `CGAL::make_mesh_3()`\endlink
```
instead of
```
\sa \link make_mesh_3() `CGAL::make_mesh_3()`\endlink
```

Reply:
let's give it a try, thank you for the suggestion